content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -48,12 +48,12 @@ Every policy set requires a configuration file named `sentinel.hcl`. This config
 The `sentinel.hcl` configuration file may contain any number of entries which look like this:
 
 ```hcl
-policy "sunny-day" {
+policy "enforce-terraform-maintenance-windows" {
```

Comment:
Would it make sense to describe the policy beforehand?

Reply:
@dylanegan great suggestion. Take a look at the latest commit and let me know if anything doesn't make sense.

Reply:
Looking great!

Reply:
You don't like sunny days? 😢 Just kidding, I get you're trying to make this more realistic. Minor, but the name is pretty long, can you do something shorter like `maintenance-windows` or `terraform-maintenance-windows`? We know you're enforcing it because it's a sentinel policy 🙃